corehq/ex-submodules/pillowtop/management/commands/sync_es_users.py

```diff
@@ -0,0 +1,23 @@
+from django.core.management.base import BaseCommand
+from corehq.dbaccessors.couchapps.all_docs import get_all_docs_with_doc_types
+from corehq.apps.users.models import CouchUser
+from corehq.apps.users.signals import update_user_in_es
+
+
+class Command(BaseCommand):
+    help = "Sync users in Elasticsearch with Couchdb (WebUser or CommCareUser)"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--doc_types',
+            required=True,
+            choices=['WebUser', 'CommCareUser'],
+            help='Specify which user doc type to sync: "WebUser" or "CommCareUser"'
+        )
+
+    def handle(self, **options):
+        doc_type = options['doc_types']
+        user_docs = get_all_docs_with_doc_types(db=CouchUser.get_db(), doc_types=[doc_type])
+
+        for user_doc in user_docs:
+            update_user_in_es(None, CouchUser.wrap_correctly(user_doc))
```

Comment:
Do we actually need to resync the users from python?  The field should already be in elasticsearch, it's just not in the mapping.  Do we have a way to tell ES to reprocess existing docs, and if so, would that be preferable?

@AmitPhulera I'd love your take on that if you could

Reply:
> This made me think if we should have documentation around changing mapping in elasticsearch and what process can be followed for different scenarios.

This would be wonderful.  Even just the summary you have here is helpful.